ote/src/clj/ote/tasks/gtfs.sql

```diff
@@ -11,6 +11,16 @@ SELECT eid.id as id, (eid."external-interface").url, eid.format[1], eid.license,
  ORDER BY "gtfs-imported" ASC LIMIT 1
    FOR UPDATE SKIP LOCKED;
 
+-- name: select-gtfs-url-for-service
+SELECT eid.id as id, (eid."external-interface").url, eid.format[1], eid.license,
+       ts."transport-operator-id" as "operator-id", ts.id as "ts-id", eid."gtfs-imported" as "last-import-date"
+  FROM "external-interface-description"  eid
```

Comment:
An extra space char on this and next row between two words :)

Reply:
fixed